tools/composer

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+mkdir -p $HOME/.composer/cache/
+
+test -t 1 && USE_TTY="--tty"
+
+docker run --rm --interactive ${USE_TTY} \
```

Comment:
Maybe if image was built locally user can be created while `docker build ...`? Uid and gid for user can be passed via build args. It will allow start container without volumes `/etc/passwd`, `/etc/group` and `$HOME`

Reply:
Hm... 
It work fine without `/etc/passwd` and `/etc/group`, I remove it)
`$HOME/.composer` required for use user host cache.

Reply:
It works without `/etc/passwd` and `/etc/group` because SSH it not
involved in fetching dependencies :)